packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -23,6 +23,9 @@ const testContext = vm.createContext({
 });
 testContext.global = testContext;
 
+const jest = {};
+jest.expect = global.expect;
```

Comment:
Is this assignment necessary here? Why wouldn't you just use `expect` instead of `jest.expect`?
Also you can remove `chai` from imports.